### PR TITLE
use keybase pub for electron downloads

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this removes our electron-download fork and makes the packager use our own mirror for electron downloads

### CODE REVIEW

**Comment by chrisnojima** [comment]
- [ ] test darwin build
- [ ] test linux build
- [ ] test windows build

cc: @zanderz @cjb

**Comment by akalin-keybase** [comment]
This is used only by the build machines (and local dev machines)?

**Comment by chrisnojima** [comment]
whatever runs it it caches it

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
we should get this in for the release

File: shared/README.md [hunk 1]
```diff
@@ -437,23 +437,15 @@ If you get this error message on trying to open the inspector:
...
 Take a look at [this repo](https://github.com/ncuillery/rn-diff), which contains branches for every version of react native. For example, this URL
 
  `https://github.com/ncuillery/rn-diff/compare/rn-0.51.0...rn-0.53.0`
 
  generates the diff between RN versions in a bare RN app. Use this to figure out if any configuration changes are needed. If the target version isn't in `rn-diff` yet, there'll usually be a fork that has it.
+
+
+### Updating `electron`
+
+We host the electron binaries in keybase.pub. If you update versions copy files from https://github.com/electron/electron/releases/ to https://keybase.pub/kbelectron/electron-download/v{version}. Make sure to get the SHASUM256.txt file also
```
Comment (buoyad) [inline shared/README.md]: Could we mention that the mirror is only used by the packager and not by developing?

**Pull request merged**